LevelUp.Pos.ProposedOrders.Tests/SplitTenderTests.cs

```diff
@@ -295,19 +301,21 @@ public void SplitTenderExample_LevelUp_Second_Discount_Remaining_Balance()
                 $"Expected: {expectedProposedOrderValues}" + Environment.NewLine +
                 $"Actual: {proposedOrderValues}");
 
-            // available discount amount $10.00
-            int availableDiscountAmount = 1000;
+            // available discount amount $5.30
+            int availableDiscountAmount = 530;
 
             // POS applies $10.00 pretax discount to check and updates subtotal and tax
```

Comment:
Line 307 needs to be modified to read $5.30 pretax discount; It still refers to $10 pretax discount.